### ISSUE TITLE
quad.3tdiag() needed

### ISSUE THREAD

**Issue by RobinHankin** [RobinHankin/emulator#1]
we have ```quad.form()``` and ```quad.tform()``` and ```quad.3form()``` and ```quad.diag()``` and ```quad.tdiag()``` but we don't have ```quad.3diag()``` or ```quad.3tdiag()```

**Comment by RobinHankin** [comment]
Stupid idea:

```
library("emulator")
library("cmvnorm")
M <- matrix(rcnorm(09),3,3)
x <- matrix(rcnorm(15),3,5)
y <- matrix(rcnorm(06),3,2)


a <- quad.3form(M,x,y)
```

Matrix ```a``` is not square.

**Comment by RobinHankin** [comment]
yes but ```quad.3form(M,x,y)  has clear meaning if ```x``` and ```y``` have the same size.  And I need this to implement  RobinHankin/lorentz#9

**Comment by RobinHankin** [comment]
fixed in 25ddd70
